application-task-test/application-task-test-docker/src/test/it/com/xwiki/task/test/ui/TaskManagerIT.java

```diff
@@ -45,7 +45,10 @@
  * @version $Id$
  * @since 2.6
  */
-@UITest
+@UITest(
+    extraJARs = {
+        "org.xwiki.platform:xwiki-platform-eventstream-store-solr"
```

Comment:
Requires a comment (I think you need it for the dependency on Mentions which uses Solr).